Index: src/com/company/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/company/Main.java	(revision afed74e9a00a170a9dc0829903dcb3f0ae03d2c8)
+++ src/com/company/Main.java	(date 1583567808145)
@@ -4,5 +4,6 @@
 
     public static void main(String[] args) {
 	// write your code here
+        System.out.println("hiiiiii");
     }
 }
